NorthstarDLL/mods/modmanager.cpp

```diff
@@ -298,11 +298,18 @@ template <ScriptContext context> auto ModConCommandCallback_Internal(std::string
 {
 	if (g_pSquirrel<context>->m_pSQVM && g_pSquirrel<context>->m_pSQVM)
 	{
-		std::vector<std::string> args;
-		args.reserve(command.ArgC());
-		for (int i = 1; i < command.ArgC(); i++)
-			args.push_back(command.Arg(i));
-		g_pSquirrel<context>->AsyncCall(name, args);
+		if (command.ArgC() == 1)
+		{
+			g_pSquirrel<context>->AsyncCall(name);
```

Comment:
Will need to ensure this doesn't break existing functions.

Reply:
What happens if the squirrel function takes an argument, but argc is 0?

Reply:
`argc` is always >= 1. I suppose what you mean is if no arg is inputted for a concommand that requires an argument? If so, it raises an error like the situation I posted.

Reply:
In that case, it would probably be a good idea to see if we can check the squirrel function arg count so we don't break existing users.

Reply:
Also, why not just pass an empty array?

Reply:
Took another look at this; it's okay.

Reply:
It's okay, nobody really uses this yet, so it shouldn't have a huge impact.